ls1intum/Artemis
# Assessment Toggle View to check Feedback

### Checklist

### Motivation and Context


Currently within the exam assessment process with 2 correction rounds it is unclear within which correction round the feedback was added. In this PR we change this. 
### Description

We add a new correction round badge to each assessment editor view. It can be toggled via a new button. The button in only viewable within the second correction round. If activated, we display the badge. It states within which correction round each feedback was added. 
Please note: 
1. It currently does not work within the Apollon editor of the modeling assessment editor. This will be fixed in a Apollon specific follow-up PR. 
2. It currently does not work within the Programming assessment editor. This will be fixed in a specific follow-up PR. It currently displays everything as part of the second correction round. 

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration -> Exams -> All assessment types. 
3. Check Text, File-upload, Modeling, Programming Exercises: 
4. Start with a first correction round (no button should be displayed) 
5. Start the second correction round (button should be displayed)
6. Test if the correct badge appears / disappears when toggling the new button. 

### Test Coverage



modelingAssessment: 78%
textAssessment: 70%
file-uploadAssessment: 92%

### Screenshots


Example Screenshots:
![Bildschirmfoto 2021-04-11 um 15 47 28](https://user-images.githubusercontent.com/42205689/114306810-5639a380-9add-11eb-944e-1259cb1147c3.png)
